instrumentation/servlet/servlet-3.0/javaagent-unit-tests/src/test/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/snippet/InjectionTest.java

```diff
@@ -0,0 +1,100 @@
+package io.opentelemetry.javaagent.instrumentation.servlet.v3_0.snippet;
+
+import static io.opentelemetry.javaagent.instrumentation.servlet.v3_0.snippet.Injection.stringInjection;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import io.opentelemetry.javaagent.bootstrap.servlet.SnippetHolder;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import org.junit.jupiter.api.Test;
+
+public class InjectionTest {
+  public String readFile(String resourceName) throws IOException {
+    InputStream is = getClass().getClassLoader().getResourceAsStream(resourceName);
+    ByteArrayOutputStream result = new ByteArrayOutputStream();
+    byte[] buffer = new byte[1024];
+    int length;
+    while ((length = is.read(buffer)) != -1) {
+      result.write(buffer, 0, length);
+    }
+    return result.toString("UTF-8");
+  }
+
+  @Test
+  public void testInjectionForStringContainHeadTag() throws IOException {
```

Comment:
(same for other tests)
```suggestion
  void testInjectionForStringContainHeadTag() throws IOException {
```